corehq/toggles.py

```diff
@@ -1514,8 +1513,8 @@ def _commtrackify(domain_name, toggle_is_enabled):
 
 CUSTOM_ICON_BADGES = StaticToggle(
     'custom_icon_badges',
-    'eNikshay: Custom Icon Badges for modules and forms',
-    TAG_CUSTOM,
+    'Custom Icon Badges for modules and forms',
+    TAG_SOLUTIONS_LIMITED,
```

Comment:
i am glad the feature i added is not getting removed entirely ❤️

Reply:
Not at all...I'm hoping this one gets into product eventually.